python/src/ExpectationSimulationAlgorithm_doc.i.in

```diff
@@ -11,70 +11,268 @@ X : :class:`~openturns.RandomVector`
 
 Notes
 -----
-The algorithm can operate on a multivariate random vector :math:`\vect{X}`.
+This algorithm estimates :math:`\Expect{\vect{X}}` using the smallest possible sample 
+size which satisfies pre-defined stopping rules. 
+It increases the sample size :math:`n` incrementally until a stopping criteria is met. 
+
+The algorithm is based on two nested loops:
+
+- the outer loop sets the number of iterations of the algorithm; 
+  this can be configured using :meth:`setMaximumOuterSampling`,
+
+- the inner loop sets the number of function calls which can be parallelized; 
+  this can be configured using :meth:`setBlockSize`.
+
+The algorithm allows to get the best possible performance on distributed 
+supercomputers and multi-core workstations. 
+For example, if the block size is equal to 100, then the sample size 
+is successively equal to 100, 200, etc... 
+Each block of evaluation of the outputs can be parallelized, which may improve the performance of the algorithm.
+We suggest to set the block size as a multiple of the number of cores. 
+
+The algorithm stops either when the maximum number of outer iterations is reached or when the precision is met. 
+The stopping criteria is based on either the coefficient of variation of the 
+sample mean (relative criterion) or the standard deviation of the sample 
+mean (absolute criterion). 
+
+Let :math:`\vect{X}\in\Rset^{n_X}` be the random vector. 
+We consider :math:`\vect{n}` independent realizations :math:`\vect{x}^{(1)}`, 
+..., :math:`\vect{x}^{(n)}\in\Rset^{n_X}` of the random vector :math:`\vect{X}`. 
+We estimate :math:`\Expect{\vect{X}}` with the sample mean:
+
+.. math::
+
+    \overline{\vect{x}} = \frac{1}{n} \sum_{i=1}^n \vect{x}^{(j)}.
+
+We estimate each component of :math:`\Var{X_i}` for :math:`i = 1, \ldots, n_X` 
+with the unbiased sample variance:
+
+.. math::
+
+    \hat{\sigma}^2_i = \frac{1}{n - 1} \sum_{i=1}^n \left(x^{(j)}_i - \overline{x}_i\right)^2
+
+for :math:`i = 1, \ldots, n_X`.
+
+The expected value of the sample mean is:
+
+.. math::
+
+    \Expect{\overline{X}_i} = \Expect{X_i}
+
+and the variance of the sample mean is:
+
+.. math::
+
+    \Var{\overline{X}_i} = \frac{\Var{X_i}}{n}.
+
+for :math:`i = 1, \ldots, n_X`, since the observations are independent.
```

Comment:
Sure?

Reply:
Yes: you have already accepted the suggestion above that inserted "since the observations are independent" before the equation. And there is a dot at the end of line 66, so the sentence should be finished.